### PR TITLE
Cockpit 307

### PR DESCRIPTION
**Pull request by jelly**
Preview: https://jelly.github.io/cockpit-project.github.io/blog/cockpit-307.html

### CODE REVIEW

File: _posts/2023-12-14-cockpit-307.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+The Storage page has been redesigned with the goal of making more things visible more of the time. The overview page now shows all storage objects in a table and allows almost all operations to be performed on them right from the table.
+
+Clicking on a row navigates to a page dedicated to the object in that row, with more information and maybe a few more actions.
+
+We will continue to improve the Storage pages, by adding filtering options to the big table and working on the actions and dialogs.
+
+![screenshot of ## storage redesign]({{ site.baseurl }}/images/307-storage-redesign.png)
+
+## Machines: Change "Add disk" default behavior
```
Comment (mvollmer) [inline _posts/2023-12-14-cockpit-307.md:30]: This should be removed, it was already released with 303.

File: _posts/2023-12-14-cockpit-307.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+State: **merged** @ <https://github.com/cockpit-project/cockpit/pull/19472>
+{:.note}
+
+The Storage page has been redesigned with the goal of making more things visible more of the time. The overview page now shows all storage objects in a table and allows almost all operations to be performed on them right from the table.
+
+Clicking on a row navigates to a page dedicated to the object in that row, with more information and maybe a few more actions.
+
+We will continue to improve the Storage pages, by adding filtering options to the big table and working on the actions and dialogs.
+
+![screenshot of ## storage redesign]({{ site.baseurl }}/images/307-storage-redesign.png)
```
Comment (mvollmer) [inline _posts/2023-12-14-cockpit-307.md:28]: This should be updated to 

image

**Review by mvollmer — CHANGES_REQUESTED** [review]

File: _posts/2023-12-14-cockpit-307.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
+---
+title: Cockpit 307
+author: jelle
```
Comment (martinpitt) [inline _posts/2023-12-14-cockpit-307.md:3]:
```suggestion
author: mvollmer
```

I think@mvollmer did most of the work here

File: _posts/2023-12-14-cockpit-307.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+date: '2023-12-14'
```
Comment (martinpitt) [inline _posts/2023-12-14-cockpit-307.md:4]:
```suggestion
date: '2023-12-15'
```

File: _posts/2023-12-14-cockpit-307.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+tags: cockpit, machines
+slug: cockpit-307
+category: release
+summary: ''
```
Comment (martinpitt) [inline _posts/2023-12-14-cockpit-307.md:8]:
```suggestion
summary: 'Storage redesign'
```

File: _posts/2023-12-14-cockpit-307.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 307:
+
+
+## Storage redesign
+
+The Storage page has been redesigned with the goal of making more things visible more of the time. The overview page now shows all storage objects in a table and allows almost all operations to be performed on them right from the table.
```
Comment (martinpitt) [inline _posts/2023-12-14-cockpit-307.md:19]: "more of the time" sounds weird. Perhaps "in all views" or just drop it?

"to be" → "can be"

**Review by martinpitt — CHANGES_REQUESTED** [review]
Thanks!

**Review by martinpitt — APPROVED** [review]
I pushed my proposed changes. Good enough for me now, thanks!

File: _posts/2023-12-14-cockpit-307.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 307:
+
+
+## Storage redesign
+
+The Storage page has been redesigned with the goal of making more things visible in all views. The overview page now shows all storage objects in a table and allows almost all operations can be performed on them right from the table.
```
Comment (allisonkarlitskaya) [inline _posts/2023-12-14-cockpit-307.md:19]: "allows almost all operations can be" doesn't work.

either

"almost all operations can be..."

or

"allows all operations to be..."

File: _posts/2023-12-14-cockpit-307.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+
+Clicking on a row navigates to a page dedicated to the object in that row, with more information and maybe a few more actions.
```
Comment (allisonkarlitskaya) [inline _posts/2023-12-14-cockpit-307.md:21]: "maybe" sounds a bit informal.  Maybe "and in some cases, additional actions".

File: _posts/2023-12-14-cockpit-307.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+We will continue to improve the Storage pages, by adding filtering options to the big table and working on the actions and dialogs.
```
[the remark this answers is not in the document]
Comment (allisonkarlitskaya) [reply _posts/2023-12-14-cockpit-307.md:23]: We called it "the storage page" above but here it's "the storage pages".

I might also try to reword the "We will continue" thing to something like "Future releases will continue this work by adding filtering..." etc.

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

**Review by allisonkarlitskaya — APPROVED** [review]
Thanks for the changes.

**Comment by martinpitt** [comment]
Thanks you two for the updates, looks nice now! :+1:

**Pull request merged**